dbms/src/Common/TiFlashMetrics.h

```diff
@@ -47,14 +47,16 @@ namespace DB
 // clang-format off
 #define APPLY_FOR_METRICS(M, F)                                                                                                                     \
     M(tiflash_coprocessor_request_count, "Total number of request", Counter, F(type_cop, {"type", "cop"}),                                          \
-        F(type_cop_executing, {"type", "cop_executing"}), F(type_batch, {"type", "batch"}),                                                         \
+        F(type_cop_executing, {"type", "cop_executing"}), F(type_cop_stream, {"type", "cop_stream"}),                                               \
+        F(type_cop_stream_executing, {"type", "cop_stream_executing"}), F(type_batch, {"type", "batch"}),                                                         \
```

Comment:
please format it.

Reply:
And the following codes.

Reply:
Update the clang-format and format this file.